monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -5859,14 +5859,10 @@ object Observable extends ObservableDeprecatedBuilders {
     * returning a new observable that generates sequences.
```

Comment:
The doc mentions that the implemenation is based on combineLatestMap2. That's not true anymore.